containers/datalab/content/run.sh

```diff
@@ -206,20 +206,20 @@ mkdir -p /content/datalab/notebooks
 
 # Fetch docs and tutorials. This should not abort startup if it fails
 {
-if [ -d /content/datalab/docs ]; then
+if [ -d /content/datalab/ga ]; then
   # The docs directory already exists, so we have to either update or initialize it as a git repository
   pushd ./
-  cd /content/datalab/docs
-  if [ -d /content/datalab/docs/.git ]; then
-    git fetch origin master; git reset --hard origin/master
+  cd /content/datalab/ga
+  if [ -d /content/datalab/ga/.git ]; then
+    git fetch origin ga; git reset --hard origin/ga
```

Comment:
That is a great idea. Would it work with Datalab CLI? Does the CLI tool support passing extra params? If so, we should certainly do that to avoid the issue above.